packages/node/src/async/index.ts

```diff
@@ -0,0 +1,17 @@
+import { NODE_VERSION } from '../nodeVersion';
+import { setDomainAsyncContextStrategy } from './domain';
+import { setHooksAsyncContextStrategy } from './hooks';
+
+/**
+ * Sets the correct async context strategy for Node.js
+ *
+ * Node.js >= 14 uses AsyncLocalStorage
+ * Node.js < 14 uses domains
+ */
+export function setNodeAsyncContextStrategy(): void {
```

Comment:
Can we get a sanity check test for this?

Reply:
Yeah, I'm surprised that there was no TypeScript or lint error from using `if (NODE_VERSION >= 14)`!